test/src/701-xattr-catalog_counters/setup_teardown

```diff
@@ -0,0 +1,27 @@
+create_nested_repo() {
+  echo "*** CREATE a fresh repository named $CVMFS_TEST_REPO with user $CVMFS_TEST_USER"
+  create_empty_repo $CVMFS_TEST_REPO $CVMFS_TEST_USER || return $?
+
+  start_transaction $CVMFS_TEST_REPO || return $?
+  mkdir -p /cvmfs/$CVMFS_TEST_REPO/foo/bar
+  touch /cvmfs/$CVMFS_TEST_REPO/root1.txt
+  touch /cvmfs/$CVMFS_TEST_REPO/root2.txt
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/.cvmfscatalog
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/foo1.txt
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/foo2.txt
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/bar/.cvmfscatalog
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/bar/bar1.txt
+  touch /cvmfs/$CVMFS_TEST_REPO/foo/bar/bar2.txt
+  publish_repo $CVMFS_TEST_REPO || return 2
+  echo "*** FINISHED creating a fresh repository named $CVMFS_TEST_REPO with user $CVMFS_TEST_USER"
+}
+
+teardown() {
```

Comment:
We typically leave the test repository up and running and let the next test destroy it. That could be changed but then it should be done consistently, I think.